src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js

```diff
@@ -230,7 +236,16 @@ class ParticipantsTable extends Component {
                 {/* Analyze in Cavatica */}
                 {/* Download */}
                 <RemoveFromCohortButton
-                  onClick={() => handleRemoveFromCohort()}
+                  onClick={() => {
+                    onRemoveFromCohort();
+                    trackTableViewAction({
+                      category: selectedRows.length > 1
+                        ? `${analyticsTracking.tableView.rows.multiRows}: ${selectedRows.length}`
+                        : analyticsTracking.tableView.rows.singleRow,
+                      action: TRACKING_EVENTS.actions.delete,
+                      label: selectedRows,
```

Comment:
for removing from cohort we should also log the active SQON _as well as_ the selected rows that were removed.

You can grab the active SQON from localstorage by 
```js
let activeSQON: localStorage.COHORT_BUILDER_FILTER_STATE.sqons[localStorage.COHORT_BUILDER_FILTER_STATE.activeIndex]
``` 
Then our label can be serialized json of 
```js 
{selectedRows, activeSQON}
```

Reply:
Now, it returns an event label as suggested.

Reply:
@liberaliscomputing  active sqon isn't getting logged here

Reply:
looking into this I discovered that the `localStorage.COHORT_BUILDER_FILTER_STATE` only gets populated when you load a saved study and it's actively updated as you query around. We are going to have to find a different way to capture `activeSQON`

@jberube thoughts on this?